test/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/ProductDetailsViewSpec.scala

```diff
@@ -244,6 +246,16 @@ class ProductDetailsViewSpec extends ViewSpec {
           .selectOptionally("div:nth-of-type(2)") shouldBe None
       }
     }
+    "the software vendor has an accessibility statement" should {
+      "display the link to the statement" in {
+        val linkText = "accessibility statement"
+        val row: Element = document(softwareVendorModelFull.copy(accessibilityStatementLink = Some(linkText)))
+          .selectNth("dl", 3)
+        row.selectHead("dt").text shouldBe ProductDetailsPage.accessibilityStatement
+        row.selectHead("dd").text shouldBe linkText
+        row.selectHead("dd").selectHead("a").getElementsByAttribute("href").text() shouldBe linkText
```

Comment:
Are you trying to get the `href` attribute value?
```suggestion
        row.selectHead("dd").selectHead("a").attr("href") shouldBe linkText
```